pyproject.toml

```diff
@@ -14,7 +14,7 @@ inflection = "^0.5.1"
 nslookup = "^1.4.0"
 pydantic = "^1.8.2"
 pytest = "^6.2.4"
-PyYAML = "^5.4.1"
+PyYAML = "!=6.0.0,!=5.4.0,!=5.4.1"
```

Comment:
@goulter: Does this mean that versions `5.4.0` and `5.4.1` are not working, but `5.4.2` or anything greater than `6.0.0` is working? Would it be better to fix the dependency to one specific stable version known to work well? Also, we could add a comment above the dependency to explain this change for future reference?

Reply:
Thanks @soufianerafik 
That sounds right to me, as far as what versions to use and what to avoid.
Yes, we probably can pin to whatever is next after 6.0.0, something like ^6.0.1. 
I guess having this not pinned would allow a safe downgrade if needed by some other dependency.